tests/run-make/link-args-order/Makefile

```diff
@@ -1,10 +0,0 @@
-# ignore-msvc
```

Comment:
Problem: yes, please restore this ignore because `ld` isn't available on Windows. Either that, or we try `llvm-lld`?